_posts/2025-02-25-cpansec-is-cna-for-perl-and-cpan.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+toc: false
+title: "CPANSec is CNA for Perl and the CPAN ecosystem"
+date: 2025-02-25 22:00:00 +0100
+tags: cpan perl cve cna
+author:  Stig Palmquist, Timothy Legge and Breno G. de Oliveira
+---
+
+The CPAN Security Group was [authorized by the CVE Program](https://www.cve.org/Media/News/item/news/2025/02/25/CPAN-Security-Group-Added-as-CNA) as a CVE Numbering Authority (CNA) on Feb 25, 2025. A CNA assigns and manages CVE identifiers for projects in their scope.
```

Comment:
minor rewording suggestion to:

As of February 25th, 2025, the CPAN Security Group is [authorized by the CVE Program](https://www.cve.org/Media/News/item/news/2025/02/25/CPAN-Security-Group-Added-as-CNA) as a CVE Numbering Authority (CNA). A CNA is responsible for assigning and managing CVE identifiers for projects within their scope.